### DISCUSSION TITLE
Tai 支持导入其他时间统计软件的数据

### DISCUSSION THREAD

**Discussion by ioqbit** [Planshit/Tai#97] [Ideas]
之前一直在使用 [ManicTime](https://www.manictime.com)，但是他更新慢、UI丑。

这些天使用了 Tai，对比之下还是相信更开源产品的潜力，愿意迁移，但又不想完全放弃之前的数据，能否添加导入数据的功能？

> ManicTime 支持数据导出

**Comment by noberumotto** [comment]
非常感谢使用和信任Tai。如果manictime支持数据导出，我会实现这个功能的，不过目前还有挺多优先级更高的问题或者需求需要处理，所以没那么快，可能得春节之后，请记得关注后续更新
